cli/src/vote.rs

```diff
@@ -214,6 +214,21 @@ impl VoteSubCommands for App<'_, '_> {
                         .long("lamports")
                         .takes_value(false)
                         .help("Display balance in lamports instead of SOL"),
+                )
+                .arg(
+                    Arg::with_name("rewards")
```

Comment:
I've been considering similar, but was going to go the route of `head`'s `-n, --lines=NUM`.  Something like `--limit-rewards NUM_EPOCHS` defaulting to the last 10 or so epochs.  I'd also kicked around a `--rewards-since EPOCH` that lists all rewards after the given EPOCH.  These feel more intuitive/ergonomic.  Was there a motivation behind the current interface that I might be missing?

Reply:
Yes, I think most users are primarily interested in the most recent rewards, so the fastest response and easiest interface is just `solana stake-account --rewards` with no option input required. I'm not totally opposed to `solana stake-account --rewards --num-epochs <NUM>`, as it does eliminate the need to know specific epoch numbers, but it wouldn't offer as much flexibility in terms of old epochs.

`--rewards-since` seems like it has the inconvenience of this PR's interface (wrt needing to know specific epoch numbers) without the benefit of clear, linear response time.

Light discussion here: https://github.com/solana-labs/solana/pull/16278#issuecomment-812601790

cc @mvines

Reply:
That said, I am planning to add the `solana inflation-rewards` subcommand as well. Perhaps that command could meet the past-epoch flexibility need, using a single `--epoch` option.

Reply:
Got it. Thanks!  Can we use more specific args to avoid getting backed into a corner with more ambiguous one-worders?  I'll propose `--with-rewards` and `--rewards-epoch`.

Reply:
Totally; thanks for the suggestions!

Reply:
In testing, I grew to strongly dislike the single, specific `--rewards-epoch`. I've reworked to `--num-rewards-epochs` here: https://github.com/solana-labs/solana/pull/16408/commits/213dcca5ff5b6dd899356c5f630dec4ec59de767
Wdyt? It's a little more complex under the hood, but still quick.